src/pages/Index/Events/index.tsx

```diff
@@ -2,103 +2,104 @@
-                            <div className="swiper-container">
-                                <div
-                                    className="swiper-wrapper slider-content"
-                                    style={{ width: events.length * 455 }}
+                <div className="content-events">
+                    <div className="events-slider" ref={swiperRef}>
+                        <i
+                            id="swiper-button-prev"
+                            className="iconfont icon-jiantou1-copy-copy-copy"
+                        />
+                        <i
+                            id="swiper-button-next"
+                            className="iconfont icon-jiantou1-copy"
+                        />
+                        <Swiper
+                            modules={[Navigation, Autoplay]}
+                            grabCursor={true}
+                            slidesPerView={3}
+                            centeredSlides={false}
+                            slidesPerGroup={1}
+                            spaceBetween={65}
+                            loop={true}
+                            autoplay={{
+                                delay: 3000,
+                                disableOnInteraction: false
+                            }}
+                            navigation={{
+                                nextEl: '#swiper-button-next',
+                                prevEl: '#swiper-button-prev'
+                            }}
+                            className="swiper-container"
+                        >
+                            {events.map(({ title, url }) => (
+                                <SwiperSlide
+                                    key={url + ''}
+                                    className="event-panel"
                                 >
-                                    {events.map(({ title, url }) => (
-                                        <div
-                                            key={url + ''}
-                                            className="swiper-slide event-panel"
-                                        >
-                                            <div>
-                                                <img src={url + ''} />
-                                                <p>{title}</p>
-                                            </div>
-                                        </div>
-                                    ))}
-                                </div>
-                                {/* <div class="swiper-scrollbar"></div> */}
-                            </div>
-                        </div>
+                                    <div>
+                                        <img src={url + ''} alt={title} />
+                                        <p style={{ color: 'white' }}>
+                                            {title}
+                                        </p>
+                                    </div>
```

Comment:
```suggestion
                                        <img src={url + ''} alt={title} />
                                        <p className="text-white">
                                            {title}
                                        </p>
```
1. 应该不需要多套一层 `<div />` 了。
2. 样式尽量用工具类：https://getbootstrap.com/docs/5.3/utilities/colors/